lib/main.js

```diff
@@ -420,6 +423,16 @@ const Hydrogen = {
     }
   },
 
+  exportToIpynb() {
+    const currentEditor = atom.workspace.getActiveTextEditor();
+    if (!currentEditor) {
+      return;
+    }
+    const filename = currentEditor.getTitle();
+    const nbName = filename.split(".")[0] + ".ipynb";
+    writeFileSync(nbName, store.notebook);
```

Comment:
I'd prefer `writeFile` here, since `writeFileSync` may block the UI for larger notebooks.

It would be great to get a notification when the file was successfully saved and a error notification otherwise.

If you really want to dig into this, you can use the [save dialog from Electron](https://electron.atom.io/docs/api/dialog/#dialogshowsavedialogbrowserwindow-options-callback) to allow the user to select a custom location. The dialog is available via
```js
import { remote } from "electron";

remote.dialog.showSaveDialog([browserWindow, ]options[, callback]);
```
You can wrap this in a Promise to use it with async/await or just do the saving in the the callback directly.

Reply:
I forget that I'm working with desktop stuff and that I have more "power" now. Thanks!!